multi-arch-builders/coreos-s390x-rhcos-builder.bu

```diff
@@ -35,12 +36,12 @@ storage:
       group:
         name: builder
   files:
-    - path: /home/core/create-secex-data.sh
-      mode: 0744
+    - path: /usr/local/bin/create-secex-data.sh
+      mode: 0755
```

Comment:
Following up on our call, YAML does support octals and the generated Ignition does have mode 493 for this file, which is correct.

Reply:
Yes, but it seems I found the root cause. Since it was maintenance window, I was running out of time and tried debugging this by rebooting the machine instead of re-installing because it was faster.
Now with more time, I managed to recreate the error with a local QEMU instance. What I found is this:
1. Creating the script with ignition under `/home/core` or `/var/home/core` -> failure
2. Moving the script on an already running system from `/var/home/core` to `/usr/local/bin`, updating the service and rebooting -> failure
3. Directly creating the script with ignition under `/usr/local/bin` -> success

So the good news is, this PR should work as is.
The bad news, I don't know if the behavior above is a bug or a feature. Since you are not really supposed to change the system after firstboot manually, it might be a feature, at least number 2.

Reply:
This is likely an SELinux issue, e.g. systemd prevented from running scripts from the home directory. You can verify this by looking for AVC denials in the journal/audit logs.